geometry_msgs/Pose current_pose
---
int32 state
geometry_msgs/Point stop_line
